docs/CONTRIBUTING.md

```diff
@@ -357,14 +380,51 @@ You can run individual unit tests by using the following syntax:
 Or by adding `.only` after a `describe` or `it` block in the test file:
 
 ```typescript
-describe.only('Extension manifest tests', () => {
-  it('should register all commands', () => {
+describe.only("Extension manifest tests", () => {
+  it("should register all commands", () => {
     // test code
   });
 });
 ```
 
-Remember to remove `.only` after running the test and before making a PR! 
+Remember to remove `.only` after running the test and before making a PR!
+
+#### Clicktesting the VS Code extension against the sidecar
+
+##### Method 1:
+
+Manually trigger the "Multi Arch Builds and Upload to GitHub Releases" promotion in the sidecar
+repo. This can be found in the Semaphore UI under the "Promotions" tab. This won't actually upload
+to a release, but will make the native executables available at the workflow's artifacts/ part of
+the Semaphore UI. Then you can continue on from step 3 of Method 2.
```

Comment:
I would leave this section out (or keep in the `ide-sidecar` repo), since external users won't have access to the Semaphore promotions unless one of us starts the `/sem-approve` flow. (And even then, I'm not sure if they would see it or be able to interact with the manual promotions at all.)